Makefile

```diff
@@ -221,3 +221,19 @@ ci-install-hypershift-private:
 		--external-dns-provider=aws \
 		--external-dns-credentials=/etc/hypershift-pool-aws-credentials/credentials \
 		--external-dns-domain-filter=service.ci.hypershift.devcluster.openshift.com
+
+.PHONY: ci-test-e2e
+ci-install-hypershift-private:
```

Comment:
copy paste mistake, target name is wrong

Reply:
yep, just noticed. fixed!